lib/main.js

```diff
@@ -5,10 +5,25 @@ var parse = require("./parse.js");
 var filters = {};
 var Lexer = parse.Lexer;
 var Parser = parse.Parser;
+var parserOptions = {
+    csp: false, // noUnsafeEval,
```

Comment:
What does that mean? Is unsafe eval our default? I would like to have a safe eval as our default.